src/main/kotlin/org/jetbrains/research/tasktracker/ui/main/panel/MainPluginPanelFactory.kt

```diff
@@ -46,10 +55,65 @@ class MainPluginPanelFactory : ToolWindowFactory {
         backButton.isVisible = isVisibleBackButton
     }
 
+    private fun welcomePage() {
```

Comment:
As far as I understand, the main idea is to provide a set of window configuration settings (which can be different for different tasks). But now, is it true, that all is hardcoded for one? Maybe you should take two tasks (not only hello world) which are different window configurations and adopt architecture for both of them?

Reply:
Yes, indeed, some things currently only work for the hello world task. I marked them with TODO comments, and I need to discuss with Sergey how we will store different configurations and retrieve them. In other words, I'll resolve this in the next PR.